generators/app/tasks/appSetup/babelConfigSetup.js

```diff
@@ -0,0 +1,34 @@
+module.exports = function babelConfigSetup() {
+  const contentBabelConfig = this.fs.read(`${this.projectName}/babel.config.js`);
+
+  const routes = `presets: ['module:metro-react-native-babel-preset'],
+  plugins: [
+    'import-glob',
+    [
+      'module-resolver',
+      {
+        root: ['./src'],
+        alias: {
+          '@app': './src/app',
+          '@assets': './src/app/assets',
+          '@components': './src/app/components',
+          '@config': './src/config',
+          '@constants': './src/constants',
+          '@lottieAssets': './src/lottieAssets',
+          '@schemas': './src/schemas',
```

Comment:
Why this? I think you added some aliases that don't exist in the bootstrap, just keep these:
```
            '@app': './src/app',
            '@components': './src/app/components',
            '@config': './src/config',
            '@constants': './src/constants',
            '@redux': './src/redux',
            '@screens': './src/app/screens',
            '@services': './src/services',
            '@utils': './src/utils'
```